x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -612,22 +603,12 @@ export const RulesList = ({
   }, []);
 
   useEffect(() => {
-    if (!setHeaderActions) return;
-
-    if (showHeaderWithoutCreateButton) {
-      setHeaderActions([<RulesListDocLink />, <RulesSettingsLink />]);
-      return;
-    }
-    if (showHeaderWithCreateButton) {
-      setHeaderActions([
-        <CreateRuleButton openFlyout={openFlyout} />,
-        <RulesSettingsLink />,
-        <RulesListDocLink />,
-      ]);
-      return;
-    }
-    setHeaderActions();
-  }, [showHeaderWithCreateButton, showHeaderWithoutCreateButton]);
+    setHeaderActions?.([
+      ...(authorizedToCreateAnyRules ? [<CreateRuleButton openFlyout={openFlyout} />] : []),
+      <RulesSettingsLink />,
```

Comment:
Just curious: Do we have a plan to share the rules' header with Observability?
It seems like a useful feature to have settings in the header (also for observability); I am wondering if we want to:
- share this logic 
- or re-implement it in observability 
- or just ignore it for some reasons in observability.

Reply:
Yea that is a good idea, I am thinking we should modularize the table even further. So something like this would be a good candidate. We don't really have anything planned as of right now but feel free to create a ticket.

Reply:
@XavierM Is there any preference about which approach we want to follow from the 3 options I mentioned above?

Reply:
we do need to have this `rulesSettingLink` in o11y rules page? I might be wrong here but my first instinct is telling me that you folks need freedom here. I will go with option 2 for now

Reply:
> we do need to have this rulesSettingLink in o11y rules page?

I don't know, that was the third option.
@grabowskit, @katrin-freihofner what do you think? Does it make sense to have this setting in the observability overview as well?